config/bootstrap.php

```diff
@@ -64,10 +63,6 @@ function ($code, $message, $file, $line, $context = null) use (&$previousHandler
     ]);
 }
 
-if (Plugin::routes('DebugKit') === false) {
-    require __DIR__ . DS . 'routes.php';
-}
```

Comment:
The Plugin class will take care of this now.

Reply:
this required and reverted here  
https://github.com/cakephp/debug_kit/commit/aabaecb032f7e91

Reply:
Yeah I goofed up.